custom/icds_reports/migrations/sql_templates/create_functions.sql

```diff
@@ -484,11 +493,17 @@ BEGIN
 	EXECUTE 'CREATE INDEX ' || quote_ident(_tablename || '_indx1') || ' ON ' || quote_ident(_tablename) || '(state_id, district_id, block_id, supervisor_id, awc_id)';
 	EXECUTE 'CREATE INDEX ' || quote_ident(_tablename || '_indx2') || ' ON ' || quote_ident(_tablename) || '(ccs_status)';
 	EXECUTE 'CREATE INDEX ' || quote_ident(_tablename || '_indx3') || ' ON ' || quote_ident(_tablename) || '(trimester)';
+    -- is a month query actually useful since the tables are partitioned by months? (general q for every table)
```

Comment:
I'm actually not sure if the month index is useful - the partitions might make it moot.